File: eCommerce/eCommerce.Services/Database/Mood.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCommerce.Services.Database
{
   public class Mood
    {
        public int MoodId { get; set; }
        public int KorisnikId { get; set; }
        public User Korisnik { get; set; }
        public StatusRaspolozenja Status { get; set; }
        public DateTime DatumEvidencije { get; set; }
        public string Opis { get; set; }
        public enum StatusRaspolozenja
        {
            Sretan = 1,
            Tuzan = 2
        }
    }
}

File: eCommerce/UI/ecommerce_mobile/lib/model/mood.dart
import 'package:ecommerce_mobile/model/user.dart';
import 'package:json_annotation/json_annotation.dart';

part 'mood.g.dart';

@JsonSerializable()
class Mood {
  int? moodId;
  int? korisnikId;
  User? korisnik;
  DateTime? datumEvidencije;
  String? opis;
   String? status;

  Mood({
    this.moodId,
    this.korisnikId,
    this.korisnik,
    this.datumEvidencije,
    this.opis,
    this.status,
  });

  factory Mood.fromJson(Map<String, dynamic> json) => _$MoodFromJson(json);

  Map<String, dynamic> toJson() => _$MoodToJson(this);
}

